src/main/scala/no/ndla/learningpathapi/validation/LanguageValidator.scala

```diff
@@ -30,7 +32,7 @@ trait LanguageValidator {
 
   object LanguageValidator {
     def validate(fieldPath: String, languageCode: String): String = {
-      languageValidator.validate(fieldPath, languageCode) match {
+      languageValidator.validate(fieldPath, languageCode, allowUnknownLanguage=false) match {
```

Comment:
Hmm. Skulle egentlig legge inn en kommentar om at jeg synes det virker litt usymmetrisk at du får lov å velge i klasse-definisjonen, men ikke her. Men det aner meg at jeg egentlig mangler litt forståelse for bruken av `trait`, `class` og `object` i denne sammenheng. Gidder du gi meg en IRL-walkthrough, @narhen ?